apollo-portal/src/main/resources/static/i18n/en.json

```
@@ -316,6 +316,8 @@
   "Index.GetCreateAppRoleFailed": "Failed to get the information of create project permission",
   "Index.Topped": "Top Successfully",
   "Index.CancelledFavorite": "Remove favorite successfully",
+  "Index.PublicProject": "Public namespace",
+  "Index.PublicTip": "You haven't created any public projects yet. You can set public projects on the project settings.",
```

I suppose the `public projects` should be `public namespaces`? And public namespaces could be created in the projects page.
```suggestion
  "Index.PublicTip": "You haven't created any public namespaces yet. You can create them in your projects.",
```